pkg/profiles/openapi.go

```diff
@@ -71,31 +75,38 @@ func swaggerToServiceProfile(swagger spec.Swagger, namespace, name, clusterDomai
 		path := path.Join(swagger.BasePath, relPath)
 		pathRegex := pathToRegex(path)
 		if item.Delete != nil {
-			spec := mkRouteSpec(path, pathRegex, http.MethodDelete, item.Delete.Responses)
+			retryable, _ := item.Delete.VendorExtensible.Extensions.GetBool("x-linkerd-retryable")
```

Comment:
We should use a string constant for "x-linkerd-retryable" rather than inlining it

Reply:
Sure. :)